src/cli/bin.js

```diff
@@ -15,6 +15,8 @@ updateNotifier({
 yargs
   .commandDir('commands')
   .demand(1)
+  // Temporary alias of {add,cat,get}
+  .commandDir('commands/files')
```

Comment:
@diasdavid Hm, probably have to change the wording. My understanding is that `files add`, `files cat` and `files get` is here to stay. `add`, `cat` and `get` is what is the aliases and the code here is aliasing add/cat/get to files add/files cat/files get but the comment doesn't seem to be informative enough.

Have any ideas on how to improve it?